content/applications/websites/website/web_design/themes.rst

```diff
@@ -0,0 +1,47 @@
+==============
+Website themes
+==============
+
+Odoo Themes provides a set of colors, fonts, and layouts to shape your website's style and configure
+various customization options. When setting up your site using the Odoo website configurator,
+selecting a theme that aligns with your desired aesthetic is imperative. If you wish to modify the
+theme later, navigate to the website builder, click the 'Edit' button, and access the 'Theme' tab.
+
+Diverse sections are at your disposal, empowering you to tailor different aspects of your site to
+meet your requirements.
+
+- :guilabel:`Colors`: The Website Builder relies on palettes composed of five colors. By default,
+  Odoo pre-selected colors based on your company logo. Defining those colors in your theme ensures
+  you stay consistent. Click :guilabel:`Color Presets` to configure your favorite color
+  combinations.
+
+- :guilabel:`Website`: From this section, you can :guilabel:`Switch Theme`,
+  :guilabel:`Add a Language`, select the :guilabel:`Page Layout`, and customize the
+  :guilabel:`Background`.
+
+  .. seealso::
+     :doc:`Translations <../configuration/translate>`
+
+- :guilabel:`Paragraph`: This section is related to the format of your text <p>.
+
+- :guilabel:`Headings`: This section is related to the format of your headings.
+
+- :guilabel:`Button`: Two types of buttons exist in Odoo: the :guilabel:`Primary Style` and the
+  :guilabel:`Secondary Style`. You can update their style from this section.
+
+  .. image::themes/buttons.png
+     :alt: two types of button in Odoo
+
+- :guilabel:`Link`: This section allows you to update the style of the hyperlinks available on your
+  website.
+
+- :guilabel:`Input Fields`: An input field is an interactive area on a graphical user interface
+  where users can enter data or provide input. You can customize these fields from this section.
+
+- :guilabel:`Advanced`: This section contains advanced features. You can switch the
+  :guilabel:`Show Header` button, inject :guilabel:`<head> and </body>` code, enter your Google map
+  custom key, change the :guilabel:`Status Colors`, and customize the :guilabel:`Grays`.
+
+  .. seealso::
+     - Google Tag Manager
+     - Google Map
```

Comment:
I think we're missing the links here

Reply:
Indeed, these pages are not created yet! I removed this seealso and took note to add the links later :)